### DISCUSSION TITLE
Cross platform manager application

### DISCUSSION THREAD

**Discussion by xafero** [86Box/86Box#3378] [Show and tell]
Hi, 

I ported "86Box Manager" by Overdoze and combined it with some parts of "86Box Manager Lite" by Insanemal and would like to offer this for everybody interested to test.

It runs on Linux, MacOS and (still) Windows.

https://github.com/xafero/86BoxManager/releases/

**Comment by Flo9696** [comment]
Don't comprress the Windows version with .tar.gz, Windows uses zip, rar or 7zip

**Reply by xafero** [reply]
Ok... new Windows release is now zip-compressed.
